pom.xml

```diff
@@ -490,7 +411,22 @@
         <configuration>
           <instructions>
             <_exportcontents>
-              io.cdap.plugin.cloud.vision.*
+              com.google.api,com.google.api.client.*,com.google.api.core,com.google.api.gax.batching,
```

Comment:
do all these need to be exported? @albertshau FYI

Reply:
removed